lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -273,7 +273,20 @@ public static function get_theme_data( $theme_support_data = array() ) {
 		if ( null === self::$theme ) {
 			$theme_json_data = self::read_json_file( self::get_file_path_from_theme( 'theme.json' ) );
 			$theme_json_data = self::translate( $theme_json_data, wp_get_theme()->get( 'TextDomain' ) );
-			self::$theme     = new WP_Theme_JSON_Gutenberg( $theme_json_data );
+			self::$theme = new WP_Theme_JSON_Gutenberg( $theme_json_data );
+
+			// If this is a child theme we want to combine the theme.json from the child with the theme.json with the parent.
+			if ( is_child_theme() ) {
+				// Get parent theme.json.
+				$parent_theme_json_data           = self::read_json_file( get_template_directory() . '/theme.json' );
+				$parent_theme_json_data           = self::translate( $parent_theme_json_data, wp_get_theme()->get( 'TextDomain' ) );
```

Comment:
```suggestion
				$parent_theme_json_data           = self::translate( $parent_theme_json_data, wp_get_theme()->parent()->get( 'TextDomain' ) );
```

I think `wp_get_theme()->get( 'TextDomain' )` here will refer to current theme - child in this case.